### PR TITLE
Fixes issue #32

### PR DESCRIPTION
**Pull request by Sai-Adarsh**
Fixes issue #32 Updated README.md

### Checklist:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you successfully ran tests with your changes locally?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Issues Solved

Fixes #{Issue Number}

-----
[View rendered README.md](https://github.com/Sai-Adarsh/HackerRank-Test-Case-Generator/blob/patch-2/README.md)

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Hi! Thanks for PR.
Make sure you have checked points in PR Template correctly.

(Powered by [GitMate.io](https://gitmate.io))

**Comment by Sai-Adarsh** [comment]
> Hi! Thanks for PR.
> Make sure you have checked points in PR Template correctly.
> 
> (Powered by [GitMate.io](https://gitmate.io))

@aashutoshrathi new PR is updated with changes

**Comment by aashutoshrathi** [comment]
You need to rebase, you are removing badge for no reason.
Also, to tick the PR template fill a small `x` in brackets like this `[x]`

**Comment by Sai-Adarsh** [comment]
> You need to rebase, you are removing badge for no reason.
> Also, to tick the PR template fill a small `x` in brackets like this `[x]`

done with the changes

**Comment by Sai-Adarsh** [comment]
> @aashutoshrathi fine ?

@aashutoshrathi ?

File: README.md [hunk 1]
```diff
@@ -19,6 +19,31 @@ Logic File | TC Generator File |
 ------------------ | ------------- |
 C / C++ / Java / Python | [TC Generator](/tc_generator/tc_gen.py) |
 
+## Install
+
+### How to Use ? 😃
+
+* Clone the repository `$ git clone https://github.com/NJACKWinterOfCode/HackerRank-Test-Case-Generator.git `.
```
[the remark this answers is not in the document]
Comment (Sai-Adarsh) [reply README.md]: > Hi, Can you please change URL to `https://github.com/aashutoshrathi/HackerRank-Test-Case-Generator.git`

DONE!

**Comment by aashutoshrathi** [comment]
Thanks

**Pull request merged**